data/beta/field/area-names.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: 'n'
+datatype: string
+field: area-names
+phase: beta 
+register: ''
```

Comment:
I am unsure if this should be blank or empty string as you have it now. But the existing data seems inconsistent. I can take a look at how the loading scripts handle this.